cpp/arcticdb/column_store/column.hpp

```diff
@@ -800,10 +815,26 @@ class Column {
         return orig_type_.value();
     }
 
+    void set_secondary_type(TypeDescriptor type) {
```

Comment:
`type` should be an rvalue shouldn't it?

Reply:
It's a trivial type, might as well just pass by value.